src/bucket/BucketController.ts

```diff
@@ -42,6 +42,11 @@ export class BucketController {
    */
   public awsSecretAccessKey: string;
 
+  /**
+   * AWS bucket path to repository. Passed in as a component binding.
+   */
+  public awsBucketPath: string;
```

Comment:
This needs a better name, how is it different from `currentFolder`? Also wouldn't it make sense for it to be a `Path` or `Folder` instead of a `string`?

Reply:
I have this as a string so that it can be passed to the angular component from the repository detail view, and then be converted into a File / Path.  I don't believe there is a way to pass in a Path / File directly.
Perhaps something like `repositoryBaseDirectory` would fit better?

Reply:
You definitely can, but I'm not sure of the exact syntax. We could use a similar name to last time, something like `awsBucketPrefix`?

Reply:
Maybe the prefix should be pointed at the root of the given users directory.  Does that impact the display.

Reply:
I currently have the code set up to point the prefix in the manner Shaun mentions.  This does currently change the display, as the entire filepath is shown in the s3commander display.

Reply:
After further discussion, we have chosen to update the naming of the variable to `awsBucketPrefix`.  We will still have to pass in the value as a `string`